src/wasm.h

```diff
@@ -90,6 +90,9 @@ struct Name : public cashew::IString {
 
 // An index in a wasm module
 typedef uint32_t Index;
```

Comment:
The mapLocals code? It should definitely be executed, unless something very strange has gone wrong. Maybe add a printf to verify it is run?

The code looks right to me, `curr` values there begin at 0 and increase. Unless I'm missing a bug?